### PR TITLE
LRDOCS-6614 Service Access Policies Update for 7.2

### PR DESCRIPTION
**Pull request by sez11a**
Please review for accuracy. Thanks! @lipusz

### CODE REVIEW

File: deployment/articles/04-securing-liferay/03-service-access-policies.markdown [hunk 1]
```diff
@@ -106,6 +107,26 @@ disallow certain API functions from being invoked, you can change the
...
+signatures appropriate to their description. For example, OAUTH2_everything.read
+matches all methods starting with `fetch`, `get`, `has`, `is`, or `search`. 
+
+**OAUTH_READ/WRITE:** These provide access to JSON web services via the OAuth
+1.0a plugin. 
+
+The default configuration opens access to all web services shipped with the
+system. Administrators should review the ones you want to use and disable the
+others. 
+
```
Comment (stian-sigvartsen) [inline deployment/articles/04-securing-liferay/03-service-access-policies.markdown:130]: I'm not sure about "opens access to all webservices shipped with the system". It sounds like by default access is open and the product is not "secure by default".

Maybe we could say something like "makes available corresponding scopes that provide access to all web services shipped with the system. The scopes must be assigned to OAuth 1 or OAuth 2 applications before they become usable."

**Review by stian-sigvartsen — COMMENTED** [review]
Looks good. I just left one comment for you to review. Thanks!

File: deployment/articles/04-securing-liferay/03-service-access-policies.markdown (hunk 1 above)
Comment (sez11a) [reply deployment/articles/04-securing-liferay/03-service-access-policies.markdown:130]: Haha. I was going for more scary. :-)

**Comment by sez11a** [comment]
Thanks, @stian-sigvartsen !

**Pull request closed without merging**